README.md

```diff
@@ -1,2 +1,203 @@
-# sdk-python
+# Vela Python SDK
+[![license](https://img.shields.io/crates/l/gl.svg)](LICENSE)
+
 Python SDK for Vela (Target's official Pipeline Automation Framework)
+
+> Vela is in active development and is a pre-release product.  Please use it at your own risk in production.
+
+This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
+
+- API version: 0.4.3
```

Comment:
Likely want to update this to match 0.6.1 since that was generated via https://github.com/go-vela/server/pull/194.

Reply:
Weird, I thought I had. Will double check what happened.